docs/photoniq/p3/view-p3-alerts.md

```diff
@@ -0,0 +1,28 @@
+---
+sidebar_position: 60
+title: View P3 Alerts
+---
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+After you have created one or more policies, you can track P3 alerts on the dashboard.
```

Comment:
What are we alerting on? We should tell the user what exactly is being tracked and might appear in this dashboard. Also, can they subscribe to the alerts somehow? I didn't see a corresponding endpoint in the API.